subjects/ai/backtesting-sp500/audit/README.md

```diff
@@ -38,7 +38,9 @@ project
 
 ###### Does the notebook contain a Histogram of average price for companies for all variables (saved the plot with the images)? This is required only for **prices.csv** data.
```

Comment:
The issue refers to this as well - it should probably be clarified in the subject that it is required to add an `histogram` with the specifics described here. Additionally, we can rephrase this, for example:

```
Does the notebook contain a Histogram that shows the average price for each company in the dataset? ...
```